Plugin/PluginBefore.php

```diff
@@ -33,7 +33,7 @@ public function beforePushButtons(
                 $result = $result[count($result) - 1];
             }
 
-            if ($result && !is_null($result["barcode"])) {
+            if ($result && $result['id']) {
```

Comment:
Er zitten in de plug-in een aantal van dit soort checks. Lijkt me goed om die overal te veranderen.?
Een voorbeeld Observer/Trackinfo.php

Reply:
Done

Reply:
@parcelpro-kalok Volgens mij ook nog in Observer/Shipment.php

Wellicht goed om ook nog even de door de code heen te zoeken op barcode en het indien van toepassing is, aan te passen.

Reply:
Oja je hebt gelijk. Ik had al in andere bestanden gekeken en daar hoeft het volgens mij niet. Bij Trackinfo.php moet je wel een barcode hebben, dus die hoef ik niet aan te passen.